packages/sdk/src/create/discovery.ts

```diff
@@ -10,7 +10,7 @@ import { CreateNodeOptions, type Libp2pComponents } from "@waku/interfaces";
 export function getPeerDiscoveries(
   enabled?: CreateNodeOptions["discovery"]
 ): ((components: Libp2pComponents) => PeerDiscovery)[] {
-  const dnsEnrTrees = [enrTree["SANDBOX"]];
+  const dnsEnrTrees = [enrTree["SANDBOX"], enrTree["TEST"]];
```

Comment:
@waku-org/js-waku returning both fleets as it is more reliable this way
sandbox was broken yesterday